### PR TITLE
feat(condo): DOMA-8846 added link to employee bot in onboarding

### PR DESCRIPTION
**Pull request by Alllex202**
## If you have a telegram bot for employees

https://github.com/open-condo-software/condo/assets/56914444/8e0ce0f7-7787-4307-88c8-9d130164a2fe

## If you don't have a telegram bot for employees

https://github.com/open-condo-software/condo/assets/56914444/27df6758-0c32-4d7d-9611-fa3ac2462634

### CODE REVIEW

**Review by nomerdvadcatpyat — APPROVED** [review]

File: apps/condo/domains/onboarding/components/TourPage/AppCards.tsx [hunk 1]
```diff
@@ -42,12 +42,26 @@ export const TechnicAppCard = () => {
...
     const TechnicAppCardText = intl.formatMessage({ id: `tour.technicAppCard.radio.text.${radioValue}` })
 
+    const showLinkToEmployeeBot = !!telegramEmployeeBotName && (radioValue === 'admin' || radioValue === 'technic')
+    const linkToEmployeeBot = useMemo(() => telegramEmployeeBotName ? (
+        <Typography.Link
+            href={`https://t.me/${telegramEmployeeBotName}`}
+            target='_blank'
+            id={`employee-telegram-bot-from-onboarding-by-technic-app-card-as-${radioValue}`}
+        >
+            {ChantBotMessage}
```
Comment (toplenboren) [inline apps/condo/domains/onboarding/components/TourPage/AppCards.tsx:60]: Hey, it seems like there are a typo here :-)

File: apps/condo/lang/en/en.json [hunk 2]
```diff
@@ -1938,7 +1939,7 @@
   "tour.pageData.billing.completed.title": "Have you worked on reducing debit debt",
   "tour.pageData.billing.completed.subtitle": "Now residents can pay the receipts, and you – follow the receipts",
   "tour.pageData.ticket.todo.title": "Optimize tickets management",
-  "tour.pageData.ticket.todo.subtitle": "When you complete all of the tasks in the list, residents will be able to submit tickets through the mobile app, and you will see them on the platform.",
+  "tour.pageData.ticket.todo.subtitle": "When you set up work with tickets, you will be able to process them faster. For example, communicate in chats about a specific tickets. Technic App{andLinkToEmployeeBot} will help you keep abreast of changes. And residents will have the opportunity to evaluate tickets.",
```
Comment (toplenboren) [inline apps/condo/lang/en/en.json:1942]: Seems like a " " is missing `Technic App{andLinkToEmployeeBot}` -> `Technic App {andLinkToEmployeeBot}`

File: apps/condo/domains/onboarding/components/TourPage/AppCards.tsx (hunk 1 above)
Comment (Alllex202) [reply apps/condo/domains/onboarding/components/TourPage/AppCards.tsx:60]: You're right, I will fix it)

File: apps/condo/lang/en/en.json (hunk 2 above)
Comment (Alllex202) [reply apps/condo/lang/en/en.json:1942]: This was intended so that there would be no double space.  You can watch preview

**Pull request merged**